src/common/messages.cpp

```diff
@@ -53,6 +53,40 @@ const std::vector<std::pair<std::string, FeeEstimateMode>>& FeeModeMap()
     return FEE_MODES;
 }
 
+std::string FeeModeInfo(const std::pair<std::string, FeeEstimateMode>& mode)
+{
+    switch (mode.second) {
+        case FeeEstimateMode::UNSET:
+            return std::string("unset: no mode set; default mode will be used.\n");
```

Comment:
This might not be a good idea because the wallet RPC `sendtoaddress` and friends default depends on whether the RPC call signals RBF or not. If it doesn’t, conservative will be used.

see

https://github.com/bitcoin/bitcoin/blob/9eb57d1ab6ea5ae642b6d6cc2e74c3046984230c/src/wallet/fees.cpp#L52

I will update the documentation to reflect this, and indicate the "unset" behaviour or `estimatesmartfee` is "economical" whereas on wallet RPC's indicate that it's economical by default but will be changed to `conservative` if the user wants did to opt into RBF.

Reply:
pushed in https://github.com/bitcoin/bitcoin/pull/30525/commits/d38a4d1f61f6b8c523f690eeb8f3a2671afaf9da, thanks for review